raiden/network/pathfinding.py

```diff
@@ -183,6 +185,28 @@ def get_pfs_iou(
         raise ServiceRequestFailed(str(e))
 
 
+def request_last_iou(
+        url: str,
+        token_network_address: typing.Union[typing.TokenNetworkAddress, typing.TokenNetworkID],
```

Comment:
Why is this a `Union`?

Reply:
At the moment we are using `TokenNetworkAddress` and `TokenNetworkID` inconsistently, which makes these union type hints necessary in a few places. The plan is to soon replace `TokenNetworkID` with `TokenNetworkAddress` completely, since they denote the same thing anyway.